### ISSUE TITLE
Calendar height with vertical orientation

### ISSUE THREAD

**Issue by AndrewBasaraba** [kizitonwose/CalendarView#145]
Hello, 
I need to create a vertical calendar that can show only one month, but I can't find a way to set the height for the calendar so that it's can be done. Tried different approaches but they did not produce any results.
So is there some way to make calendar height "wrap content" but only show one month?

**Comment by kizitonwose** [comment]
The problem here is that what would be the definition of one month's height? A month can vary in height when you consider that some have 4 rows, some 5 and others even 6 rows. The header and footer may also differ so when in vertical mode, which month's total height should be used? 
If you want to restrict the height of the view entirely, you can always set a specific height instead of using `wrap_content` or `match_parent`.

Example:

```xml
<com.kizitonwose.calendarview.CalendarView
	android:id="@+id/calendarView"
	android:layout_width="match_parent"
	android:layout_height="500dp"/> <!-- 500dp height-->
```

**Comment by AndrewBasaraba** [comment]
Let me explain a bit more. Under calendar i have a recyclerView with some elements. Calendar have static header, and rows with weeks. So if the calendar have 6 weeks, then I want it to occupy a height of 6 weeks + a header. Same for 5 weeks.
For horizontal calendar, if i make height "wrap content" it will occupy height it needs to show header+6 or 5 weeks. But if i make "wrap content" for vertical, its just occupy whole screen, and my recycler will move under screen. So is there some solution, to let calendar decide how much space it needs to show one month(header + 6 or 5 weeks) in vertical mode.
I tried to change height in monthScrollListener, but it didnt work as i planned, also i set specific height, 320dp, with 5weeks its ok, but with 6, it only show 5 and half, and monthScrollListener doesnt work correctly,

**Comment by kizitonwose** [comment]
You are right. This behavior is now fixed in [0.3.4](https://github.com/kizitonwose/CalendarView/releases/tag/0.3.4). The only catch is that the calendar scroll mode has to be `paged` so it knows when to adjust its height. There is also a new `cv_wrappedPageHeightAnimationDuration` for adjusting the height change animation duration.

**Comment by AndrewBasaraba** [comment]
Thank you, everything works as it should now.
